### DISCUSSION TITLE
Refresh Accessory Names?

### DISCUSSION THREAD

**Discussion by dmurchie** [iklein99/homebridge-smartthings#118] [Q&A] [answered]
Anyone know a way to refresh accessory names in HomeKit based on name changes made in SmartThings? 

The plug-in seems to be holding onto the name when added. Tried restarting, no change.

**Comment by iklein99** [comment, marked as answer]
You can delete the ~/.homebridge/accessories/cachedAccessories file and homebridge will reload them.  BUT, the rooms will all be reset, so you may not want to do that.

**Reply by dmurchie** [reply]
To re-name or to re-room? That is the question...

**Comment by coryabooth** [comment]
In my Homebridge, this is built into the UI.
Triple-Dots - top right...
Homebridge Settings...
Remove Single Cached Accessory...

There is also the "Unpair Bridges / Cameras / TVs / External Accessories"
For removing devices you added with a code...
